### PR TITLE
Port Stretch Description to ROS 2

### PR DESCRIPTION
**Pull request by DLu**
Note: The only reason the xacro had to be changed is because of a bug where the launch file attempted to parse it as a yaml file.

### CODE REVIEW

File: stretch_description/launch/display.launch.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+from launch import LaunchDescription
+from launch.substitutions import Command
+
+from launch_ros.actions import Node
+
+
+def generate_launch_description():
+    stretch_description_path = get_package_share_path('stretch_description')
+    robot_description_content = Command(['xacro ', str(stretch_description_path / 'urdf' / 'stretch_main.xacro')])
```
Comment (hello-binit) [inline stretch_description/launch/display.launch.py]: Could you change 'stretch_main.xacro' to 'stretch_description.xacro`? The latter includes the lidar, depth camera, end effector, and respeaker.

File: stretch_description/launch/display.launch.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    joint_state_publisher_gui_node = Node(
+        package='joint_state_publisher_gui',
+        executable='joint_state_publisher_gui'
```
Comment (hello-binit) [inline stretch_description/launch/display.launch.py]: The joints starting at zeros visualizes the robot intersecting with itself. We can use the 'zeros' parameter to change the stow config:

```python
joint_state_publisher_gui_node = Node(
    package='joint_state_publisher_gui',
    executable='joint_state_publisher_gui',
    parameters=[{'zeros.joint_lift': 0.2, 'zeros.joint_wrist_yaw': 3.4}]
)
```

**Review by hello-binit — CHANGES_REQUESTED** [review]
Thanks David, looks mostly good. Left two pieces of feedback.

**Comment by DLu** [comment]
Excellent points! Fixed.

**Review by hello-binit — APPROVED** [review]

**Comment by hello-binit** [comment]
It's in! Thanks!

**Pull request merged**